### ISSUE TITLE
[BUG] https://www.mccarthy.com/craft/search?jobviteiframe=job%2FoYD3tfwR has missing content for the page, even with the waitFor parameter set to True

### ISSUE THREAD

**Issue by calebpeffer** [mendableai/firecrawl#509]
When using the /scrape endpoint on https://www.mccarthy.com/craft/search?jobviteiframe=job%2FoYD3tfwR, the job description information doesn't appear on the page.

[] notify will on crisp once completed

**Comment by rafaelsideguide** [comment]
@tomkosm pretty interesting case here

I did the following request:
```js
{
  "url": "https://www.mccarthy.com/craft/search?jobviteiframe=job%2FoYD3tfwR",
  "actions": [
    {
      "type": "wait",
      "milliseconds": 5000
    },
    {
      "type": "scroll",
      "direction": "down",
      "amount": 15000
    },
    {
      "type": "screenshot"
    }
  ]
}
```

And still got a screenshot at the top of the page: `https://service.firecrawl.dev/storage/v1/object/public/media/screenshot-c7c8c594-cb74-4810-842c-d499bc76e452.png`. Not sure if scroll doesn't work for this specific page or what's going on. Can you take a look at it?

**Comment by rafaelsideguide** [comment]
it's an iframe 😰 https://github.com/mendableai/firecrawl/issues/136

**Comment by rafaelsideguide** [comment]
Solved by #855
